peps/pep-0770.rst

```diff
@@ -406,18 +379,73 @@ PyPI and other indices MAY validate the contents of SBOM documents specified by
 this PEP, but MUST NOT validate or reject data for unknown
 SBOM standards, versions, or fields.
 
+.. _770-backwards-compat:
+
 Backwards Compatibility
 =======================
 
-There are no backwards compatibility concerns for this PEP.
-
-The changes to Python package Core Metadata and ``pyproject.toml`` are
-only additive, this PEP doesn't change the behavior of any existing fields.
-
-Tools which are processing Python packages can use the ``Sbom-File`` core
-metadata field to clearly delineate between packages which include SBOM
-documents that implement this PEP (and thus have more requirements) and
-packages which include SBOM documents before this PEP was authored.
+Reserved ``.dist-info`` subdirectories registry
+-----------------------------------------------
+
+The new registry of reserved ``.dist-info`` subdirectories represents
+a new reservation that wasn't previously documented, thus has the potential to
+break assumptions being made by already existing tools.
+
+To check what ``.dist-info`` subdirectory names are in use today
+a query across
+`all files in package archives on PyPI <https://sethmlarson.dev/security-developer-in-residence-weekly-report-18>`__
+was executed:
+
+.. code-block:: sql
+
+    SELECT (
+      regexp_extract(archive_path, '.*\.dist-info/([^/]+)/', 1) AS dirname,
+      COUNT(DISTINCT project_name) AS projects
+    )
+    FROM '*.parquet'
+    WHERE archive_path LIKE '%.dist-info/%/%'
+    GROUP BY dirname ORDER BY projects DESC;
+
+Note that this only includes records for
+*files* and thus won't return results for empty directories. Empty directories
+being pervasively used and somehow load-bearing is unlikely, so is an accepted
+risk of using this method. This query yielded the following results:
+
+.. table::
```

Comment:
Likewise, you don't need the `table` directive here, and you can de-dent the table itself

Reply:
https://github.com/python/peps/pull/4283/commits/f0258e8e81db992953ee2424c7af1600b34070cb